layouts/partials/posts.html

```diff
@@ -14,7 +14,7 @@
                   {{ end }}
                   {{ end }}
                   <div class="absolute inset-0 flex justify-center items-center">
-                      <div class="px-3 block text-center mt-2 text-2xl font-black text-white hover:text-primary-600 dark:hover:text-primary-500">{{.Title}}</div>
+                      <div class="px-3 block text-center mt-2 text-2xl font-black text-white hover:text-primary-600 dark:hover:text-primary-500" style="background-color: rgb(100 100 100 / 40%); max-width: 80%;">{{.Title}}</div>
```

Comment:
The goal is to add a black background for the title so that the white text remains readable regardless of the background image used.
I did not want to change the `style.css` because it seems to be automatically updated. That's a bit hacky to put that here but I don't really know where to put it.